src/BlockchainTestsFiller/ValidBlocks/bcStateTests/selfdestructBalanceFiller.yml

```diff
@@ -0,0 +1,181 @@
+    coinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    difficulty: 0
+    extraData: 0x42
+    gasLimit: F00000000000
+    gasUsed: 0
+    mixHash: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+    nonce: 0x0102030405060708
+    number: 0
+    parentHash: 0x0000000000000000000000000000000000000000000000000000000000000000
+    receiptTrie: 0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421
+    stateRoot: 0xf99eb1626cfa6db435c0836235942d7ccaa935f1ae247d3f1c21e495685f903a
+    timestamp: 0x54c98c81
+    transactionsTrie: 0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421
+    uncleHash: 0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347
+    baseFeePerGas: 10
+
+
+  _info:
+    comment: Ori Pomerantz     qbzzt1@gmail.com
+
+  pre:
+
+    # The "user" account
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: F000000000000000
+      code: 0x
+      nonce: 1
+      storage: {}
+
+    # Suicidal contract
+    000000000000000000000000000000000000dead:
+      balance: 10
+      code: |
+        :yul {
+          selfdestruct(0x1000)
+        }
+      nonce: 1
+      storage: {}
+    
+
+    ccccccccccccccccccccccccccccccccccccccc0:
+      balance: 10
+      code: |
+       :yul {
+          // Store initial balance
+          sstore(0, balance(0xDEAD))
+
+          // Call 0xDEAD. It will commit suicide
+          let delme := call(gas(), 0xDEAD, 1, 0,0, 0,0)
+
+          // Call result
+          sstore(1, delme)
+
+          // Balance of 0xDEAD, should be zero
+          sstore(2, balance(0xDEAD))
+          
+          // Balance 0f 0x1000, should be eleven
+          sstore(3, balance(0x1000))
+
+
+          // Our balance
+          sstore(4, balance(0xccccccccccccccccccccccccccccccccccccccc0))
+
+          // Call 0xDEAD. No code, this just transfers 1 wei
+          delme := call(gas(), 0xDEAD, 1, 0,0, 0,0)
+
+          // Call result
+          sstore(5, delme)
+
+          // Balance of 0xDEAD, should be 1
+          sstore(6, balance(0xDEAD))
+
+          // Our balance, should be lower
+          sstore(7, balance(0xccccccccccccccccccccccccccccccccccccccc0))
+
+          
+       }        
+      nonce: 1
+      storage:
+        0: 0x60A7
+        1: 0x60A7
+        2: 0x60A7
+        3: 0x60A7
+        4: 0x60A7
+        5: 0x60A7
+        6: 0x60A7
+        7: 0x60A7
+
+
+    ccccccccccccccccccccccccccccccccccccccc1:
+      balance: 10
+      code: |
+       :yul {
+          // Should be zero
+          sstore(0, balance(0xdead))
+          // 0xDEAD is already dead, but this call will give it 1 wei and
+          // "resurrect" it
```

Comment:
Check the ressurected contract nonce and code

Reply:
Done